charts/plugin-fees/templates/fees/serviceaccount.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.fees.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "plugin-fees.fullname" . }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)